modules/appeals_api/app/swagger/appeals_api/v1/schemas/notice_of_disagreements.rb

```diff
@@ -0,0 +1,330 @@
+# frozen_string_literal: true
+
+module AppealsApi::V1
+  module Schemas
+    class NoticeOfDisagreements
+      include Swagger::Blocks
+
+      swagger_component do
+        schema :nodCreateInput do
+          key :required, %i[type attributes]
+          key :description, 'Form 10182 with minimum required to establish.'
+
+          property :data do
+            key :type, :object
+            property :type do
+              key :type, :string
+              key :example, 'noticeOfDisagreement'
+              key :description, 'Required by JSON API standard'
+            end
+
+            property :attributes do
+              key :type, :object
+              key :description, 'Required by JSON API standard'
+              key :required, %i[veteran phone boardReviewOption timezone]
+
+              property :veteran do
+                key :type, :object
+                key :description, 'Veteran Object being submitted in appeal'
+                key :required, %i[homeless address phone emailAddressText]
```

Comment:
As of PR #5545, the veteran's address, phone, and email are no longer a part of the schema. Supplying them will now result in an error.